<?xml version="1.0"?>
<launch>
  <!--include file="$(find youbot_arm_joints)/launch/bringup.launch" /-->
  <node pkg="youbot_arm_ik" type="youbot_arm_ik_server" name="youbot_arm_ik_server" output="screen">
  </node>
</launch>

